[2022-04-13 22:36:06,687] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:24+00:00 [queued]>
[2022-04-13 22:36:06,727] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:24+00:00 [queued]>
[2022-04-13 22:36:06,728] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:06,729] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:36:06,729] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:06,802] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-04-02 06:00:24+00:00
[2022-04-13 22:36:06,833] {standard_task_runner.py:52} INFO - Started process 11286 to run task
[2022-04-13 22:36:06,836] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-04-02T06:00:24+00:00', '--job-id', '1946', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpf4gtvw4n', '--error-file', '/tmp/tmp52fe62xz']
[2022-04-13 22:36:06,860] {standard_task_runner.py:77} INFO - Job 1946: Subtask ingest
[2022-04-13 22:36:07,265] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:24+00:00 [running]> on host 13db72185839
[2022-04-13 22:36:07,518] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-02T06:00:24+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-02T06:00:24+00:00
[2022-04-13 22:36:07,520] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:36:07,524] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:36:07,572] {subprocess.py:85} INFO - Output:
[2022-04-13 22:36:07,579] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:36:07,580] {subprocess.py:89} INFO - dags
[2022-04-13 22:36:07,581] {subprocess.py:89} INFO - logs
[2022-04-13 22:36:07,582] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:36:07,582] {subprocess.py:89} INFO - output_2022-03.csv
[2022-04-13 22:36:07,583] {subprocess.py:89} INFO - output_2022-04.csv
[2022-04-13 22:36:07,585] {subprocess.py:89} INFO - plugins
[2022-04-13 22:36:07,587] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:36:07,588] {subprocess.py:89} INFO - scripts
[2022-04-13 22:36:07,590] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:36:07,591] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:36:07,682] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220402T060024, start_date=20220413T223606, end_date=20220413T223607
[2022-04-13 22:36:07,796] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:36:08,006] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
